src/server/plugins/engine/plugin.ts

```diff
@@ -267,6 +160,14 @@ export const plugin = {
           )
         }
 
+        const emailAddress = definition.outputEmail
+
+        if (!emailAddress) {
```

Comment:
Sorry, I know I had a comment related to this before.... can we make this fail only for live forms? Previewing a form without an email is something the content team will want to do regularly, otherwise they'll never be able to see forms without our input.

```suggestion
        if (!emailAddress && !isPreview) {
```

Reply:
Spoke with Dave on slack, change pushed.